src/main/java/seedu/wife/commons/util/StringUtil.java

```diff
@@ -77,10 +98,41 @@ public static String capitalizeString(String s) {
 
         for (int i = 0; i < words.length; i++) {
             words[i] = words[i].substring(0, 1).toUpperCase() + words[i].substring(1);
```

Comment:
This seems ok but would be better to have it as such `words[i].substring(0, 1).toUpperCase() + words[i].substring(1).toLowerCase();`. This is so that when user inputs "rAnDoM TAG", it will be "Random Tag" instead of "RAnDoM TAG"